src/icrar/leap-accelerate/tests/algorithm/PhaseRotateTests.cc

```diff
@@ -97,14 +190,14 @@ namespace icrar
             }
             catch(...)
             {
-                FAIL() << "Excpected std::invalid_argument";
+                FAIL() << "Expected std::invalid_argument";
             }
         }
     };
 
     TEST_F(PhaseRotateTests, DISABLED_PhaseRotateTestCpu) { PhaseRotateTest(false); }
     TEST_F(PhaseRotateTests, DISABLED_PhaseRotateTestCuda) { PhaseRotateTest(true); }
-    TEST_F(PhaseRotateTests, DISABLED_RotateVisibilitiesTestCpu) { RotateVisibilitiesTest(false); }
+    TEST_F(PhaseRotateTests, RotateVisibilitiesTestCpu) { RotateVisibilitiesTest(false); }
```

Comment:
Currently only this test is enabled but subsequent pull requests are up to enable the others